src/utils/helpers/structure/addReviewResponses.ts

```diff
@@ -0,0 +1,31 @@
+import { ReviewResponse } from '../../generated/graphql'
+import { FullStructure, SetReviewResponseOnElement } from '../../types'
+import getGroupedReviewResponses from './getGroupedReviewResponses'
+
+const addReviewResponses = (
+  structure: FullStructure,
+  sortedReviewResponses: ReviewResponse[],
+  setLatestResponseOnElement: SetReviewResponseOnElement,
+  setPreviousResponseOnElement: SetReviewResponseOnElement
+) => {
+  const { elementsById } = structure
+
+  if (!elementsById) return structure
+
+  const groupedReviewResponses = getGroupedReviewResponses(sortedReviewResponses)
```

Comment:
Just noticed this TODO on `getGroupedReviewResponses` can probably be implemented now:
https://github.com/openmsupply/application-manager-web-app/blob/d5f9df6de8800f060b2ca952e753df12549b28b7/src/utils/helpers/structure/getGroupedReviewResponses.ts#L11-L12

Reply:
f2d4010c2f74c4608b65ab02752998015747ec95